### DISCUSSION TITLE
Delete

### DISCUSSION THREAD

**Discussion by Yumega** [music-assistant/support#1324] [Q&A] [answered]
Delete

**Comment by OzGav** [comment]
Have you selected the Show to sidebar option in the addon UI

**Reply by Yumega** [reply]
Sorry. I can’t find the option,where is it

**Reply by OzGav** [reply]
It’s in the same place as every other addon. Fourth option down
![IMG_0778](https://github.com/music-assistant/hass-music-assistant/assets/19848947/49a2ab8d-b594-4169-acec-cfdb321e0ce4)

**Reply by Yumega** [reply]
Sorry,I'm a Home assistant new user from yesterday. I don't have an addon UI like this, I need make a research

**Reply by OzGav** [reply]
Go to SETTINGS>>ADDONS

**Reply by Yumega** [reply]
I even don't have the addons options

**Reply by OzGav** [reply]
Ok are you running that in docker?

**Reply by Yumega** [reply]
Yes, in docker

**Reply by OzGav** [reply, marked as answer]
Ok to have that work you will need to go to your HA host and you can use an iframe https://www.home-assistant.io/integrations/panel_iframe/ and point that to YOUR-MA-SERVER-IP-ADDRESS:8095 

That should get you to what you are after. If you have the ability run Home Assistant Operating System then that will makes things far simpler for you

**Reply by Yumega** [reply]
Thank you very much, the sidebar has music assistant now, and it works excellent, can play music from local disk.
I added one speaker on HA dashbaord, click browse media--- radio stations(from tunein radio) have items, but the Artists and albums sections don't have any items,can't read local disk?

**Reply by OzGav** [reply]
The HA interface is limited. You should access the MA interface to select your music
